src/common/Executor.cpp

```diff
@@ -158,15 +165,18 @@ void Executor::setupSuite()
     run_params.setInvalidExcludeFeatureInput(invalid);
 
     //
```

Comment:
TODO:  Rich, David B., there is functionality we have not yet implemented in Kokkos, but that is important for RAJAPerf Suite.  Starting at line 167, and ending at ~ line 342 in Executor.cpp, we have commented this functionality.  Rich, David B. suggested that I work with you on how to handle integrating this (currently) commented functionality with Kokkos design (i.e., dynamic addition of new kernels and kernel groups).